backend/Dockerfile.cacheserver

```diff
@@ -31,7 +31,11 @@ RUN go-licenses csv ./backend/src/cache > /tmp/licenses.csv && \
     diff /tmp/licenses.csv backend/third_party_licenses/cache_server.csv && \
     go-licenses save ./backend/src/cache --save_path /tmp/NOTICES
 
-FROM alpine:3.8
+FROM alpine:3
```

Comment:
I would rather prefer to have the exact version instead of generic `3` here. WDYT?

Reply:
@gkcalat all these CVEs are coming from `alpine:3.8` and the latest version from alpine is `3.17`. https://hub.docker.com/_/alpine

If we move to 3.17, it will fix the issues, but a few months down the line, we will be doing this same exercise, as they might find new vulnerabilities in 3.17.

The most ideal solution is to use the tag `latest` or `stable`. I see some examples within this git repo, where we are using these tags. 
https://github.com/kubeflow/pipelines/blob/4d58f7fddc7ba2805613039f1cac1aa5c98cba8d/backend/Dockerfile.viewercontroller#L32
https://github.com/kubeflow/pipelines/blob/4d58f7fddc7ba2805613039f1cac1aa5c98cba8d/backend/Dockerfile#L59
 

I can switch these images to latest like in `viewercontroller` example if you like. Let me know.

Reply:
Hi @deepk2u!

I apologize for the late response.

Resolving CVE is orthogonal to the base image tag. Even with the `:latest` we will need to release a patch version of all containers. The drawback is the possibility of having unexpected change in behavior and broken tests.

The examples you pointed out should be changed too. If you are interested, you can pin them too in this or a separate PR.

Reply:
@gkcalat I updated the images to 3.17. I will make changes to other docker files in a separate PR.